File: ympa_fsharp_functions_server/src/model/GetSuggestedOfferMappingsResponse.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.ApiResponseStatusType
open ympa_fsharp_functions_server.Model.GetSuggestedOfferMappingsResultDTO

module GetSuggestedOfferMappingsResponse =

  //#region GetSuggestedOfferMappingsResponse

  [<CLIMutable>]
  type GetSuggestedOfferMappingsResponse = {
    [<JsonProperty(PropertyName = "status")>]
    Status : ApiResponseStatusType;
    [<JsonProperty(PropertyName = "result")>]
    Result : GetSuggestedOfferMappingsResultDTO;
  }

  //#endregion

File: src/gatling/scala/org/openapitools/client/model/GetSuggestedOfferMappingsResponse.scala

package org.openapitools.client.model


case class GetSuggestedOfferMappingsResponse (
    _status: Option[ApiResponseStatusType],
    _result: Option[GetSuggestedOfferMappingsResultDTO]
)
object GetSuggestedOfferMappingsResponse {
    def toStringBody(var_status: Object, var_result: Object) =
        s"""
        | {
        | "status":$var_status,"result":$var_result
        | }
        """.stripMargin
}
